src/pages/Home/home.js

```diff
@@ -1,43 +1,156 @@
 import axios from "axios";
 import React from "react";
+import { Link } from "react-router-dom";
+import { Container } from "./home.styles";
+import { Card } from "./home.styles";
 
 export class Home extends React.Component {
   state = {
-    data: {},
+    horror: {},
+    romance: {},
+    mystery: {},
+    nonfiction: {},
+    history: {},
```

Comment:
data : null,
etc.

Using an empty object will give you a headache of unexpected errors trust me